design/token.go

```diff
@@ -218,12 +218,14 @@ var OauthToken = a.MediaType("application/vnd.oauthtoken+json", func() {
 	a.Description("Oauth 2.0 token payload")
 	a.Attributes(func() {
 		a.Attribute("access_token", d.String, "Access token")
-		a.Attribute("expiry", d.String, "Expiry")
+		a.Attribute("expires_in", d.String, "Access token expires in seconds")
+		a.Attribute("expiry", d.String, "The same as expires_in.")
```

Comment:
Can't we just get rid of `expiry` ? `grant_type=refresh_token` and `grant_type=authorization_code` are not used by any other services so far and in `grant_type=client_credentials` we are not passing `expires_in` or `expiry` (why?).

Reply:
Fair point. 
What does the spec say? 

grant_type=client_credentials doesn't have an expiry because it is not a jwt, plus if needed we will simply change the credentials everywhere.

Reply:
But in future, this will be consumed in other places too ( in the platform UI for example ), so it'll be nice to have Support for all dialects.

Reply:
As per specs `expires_in` in not required, but recommended https://tools.ietf.org/html/rfc6749#section-5.1

Reply:
OAuth2.0 and OpenID Connect both uses `expires_in`. `expiry` might have come from an unreliable reference or from docs of some external provider who doesn't follow specs properly, in which case we don't need to support `expiry` any more. Also the word `expiry` mislead you for considering it as a time-stamp.